int32 camera_status
int32 controller_status